File: 100-Days-Series/Day 85/SumOfSquares.java
public class SumOfSquares {
    // Main Method 
    public static void main(String [] args){
        int n;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        // Input : value of n 
        System.out.print("Enter the value of N : ");
        n=sc.nextInt();

        // Formula of calculate the sum of squares 
        int sum = ( n * ( n + 1 ) * ( 2 * n + 1 ) ) / 6;

        // Output : the sum of series
        System.out.println("Sum of Series : "+sum);
    }    
}

File: 100-Days-Series/Day 85/SumOfSquares.cpp
#include<iostream>
using namespace std;
// Let's Start Here
int main(){
    int n;
    // Input : value of N
    cout<<"\nEnter  the value of N : ";
    cin>>n;

    // Formula to calculate the sum of squares
    int sum = ( n * ( n + 1 ) * ( 2 * n + 1 ) ) / 6;

    // Output : sum of the series
    cout<<"\nSum of the series : "<<sum<<endl;
    return 0;

}